packages/react/src/components/Table/TableHeader.tsx

```diff
@@ -1,21 +1,53 @@
 import React, { ThHTMLAttributes } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Icon from '../Icon';
+
+// these match aria-sort's values
+type SortDirection = 'ascending' | 'descending' | 'none';
 
 interface TableHeaderProps extends ThHTMLAttributes<HTMLTableCellElement> {
   children: React.ReactNode;
+  sortDir?: SortDirection;
```

Comment:
I'm not sure what we have done elsewhere/if we have a standard, but my preference is to avoid abbreviations generally (`sortDir` -> `sortDirection`)